packages/neon/neon_notifications/lib/neon_notifications.dart

```diff
@@ -56,9 +56,9 @@ class NotificationsApp extends AppImplementation<NotificationsBloc, Notification
   BehaviorSubject<int> getUnreadCounter(final NotificationsBloc bloc) => bloc.unreadCounter;
 
   @override
-  (bool? supported, String? minimumVersion) isSupported(
+  VersionSupported<String?> isSupported(
     final Account account,
     final core.OcsGetCapabilitiesResponseApplicationJson_Ocs_Data capabilities,
   ) =>
-      const (null, null);
+      const (supported: true, minimumVersion: null);
```

Comment:
I feel like this should still be null. We don't know if the app is actually supported because that depends on the core. Thus saying it is, is not actually true. Null means we don't know and I think we should keep it that way

Reply:
Then I'd rather make the entire check return null. When a check is provided it shouldn't be null able.

Reply:
Yep that's better. Without named records this was annoying with deconstructing the records if the whole record was null, but that isn't a problem anymore

Reply:
Ah I remember :+1: